client/js/components/ReminderSwitch.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
```

Comment:
all the components should be in their component dir with index file and styles.js file. Please make the changes.